src/NixFromNpm/Options.hs

```diff
@@ -205,7 +205,7 @@ validateOptions opts@(RawOptions{..}) = do
   tokensCommandLine <- parseNpmTokens roNpmTokens
   npmTokensEnv <- getNpmTokens
   return $ NixFromNpmOptions {
-    nfnoOutputPath = outputPath,
+    nfnoOutputPath = collapse outputPath,
```

Comment:
I'm not familiar with the `collapse` function; what's going on here?

Reply:
[`Filesystem.Path.collapse`](https://hackage.haskell.org/package/system-filepath-0.4.13.4/docs/Filesystem-Path.html#v:collapse) will resolve the interstitial special directory path segments `.` and `..` to a collapsed absolute path.

```haskell
>>> :set -XOverloadedStrings
>>> import qualified Filesystem.Path
>>> import qualified Filesystem.Path.CurrentOS
>>> Filesystem.Path.collapse (Filesystem.Path.CurrentOS.fromText "/home/parnell/./Desktop/../")
FilePath "/home/parnell/"
```

Most tools will handle the special dirs but it's good practice I think to collapse them to the resolved absolute path when possible since some tools won't handle it well. I noticed that `nixfromnpm` wasn't collapsing dir paths when the Nix build script was failing to run `wrapProgram` on the scripts within the `tools/` directory. I had invoked `nixfromnpm` like this:

```shell
[nix-shell:~/Development/work]$ nixfromnpm --registry <private_registry> -p pulp -o ./
```

Reply:
Ah this is called `normpath` in python. I agree it's a good practice. :+1: